aggregator/src/aggregator.rs

```diff
@@ -851,6 +854,41 @@ impl<C: Clock> TaskAggregator<C> {
     }
 }
 
+/// Used by the aggregation job initialization handler to represent initialization of a report
+/// share.
+#[derive(Clone, Debug)]
+struct ReportShareData<const L: usize, A>
```

Comment:
I really wish this struct could be defined closer to the two methods on `VdafOps` that use it, but Rust won't allow you to define a struct inside another `impl`. You can only do that inside a method.

I also considered taking those parts of `VdafOps` that pertain to handling `AggregationJobInitReq` and moving them into a new module `aggregator/src/aggregator/aggregation_job_init_handler.rs` that would define `struct AggregationJobInitHandler`. I think this would help with navigating Janus (`aggregator.rs` is already nearly 10,000 lines), but it would make a mess of this diff, so I'm not doing it now.

Reply:
FWIW, we could split the `impl` block in two, to group related code together now with a minimally invasive change.

Reply:
That's a good idea, thank you!